addons/website/static/src/less/website.edit_mode.less

```diff
@@ -21,6 +21,12 @@
 
 .o_editable {
     &:not(:empty), &[data-oe-type] {
+        &[data-oe-type="datetime"], &[data-oe-type="date"] {
+            &.o_editable_date_field_linked {
+                box-shadow: @odoo-brand-primary 0 0 5px 2px inset;
+                background-color: ligthen(@odoo-brand-primary, 20%);
+            }
+        }
```

Comment:
Can't you just combine with the line below? I don't see the background-color at all anyway. In fact you made a typo in lighten and the background-color is not even used.